content/en/docs/instrumentation/go/getting-started.md

```diff
@@ -1,681 +1,1425 @@
 ---
 title: Getting Started
 weight: 10
-cSpell:ignore: chan codebases fscanf println stdouttrace strconv struct
+# prettier-ignore
+cSpell:ignore: chan fatalln funcs intn itoa khtml otelhttp rolldice stdouttrace strconv
 ---
 
-Welcome to the OpenTelemetry for Go getting started guide! This guide will walk
-you through the basic steps in installing, instrumenting with, configuring, and
-exporting data from OpenTelemetry. Before you get started, be sure to have Go
-1.16 or newer installed.
+This page will show you how to get started with OpenTelemetry in Go.
 
-Understanding how a system is functioning when it is failing or having issues is
-critical to resolving those issues. One strategy to understand this is with
-tracing. This guide shows how the OpenTelemetry Go project can be used to trace
-an example application. You will start with an application that computes
-Fibonacci numbers for users, and from there you will add instrumentation to
-produce tracing telemetry with OpenTelemetry Go.
+You will learn how you can instrument a simple application automatically, in
+such a way that [traces][] and [metrics][] are emitted to the console.
```

Comment:
```suggestion
You will learn how you can instrument a simple application using
instrumentation libraries and custom instrumentation with
OpenTelemetry APIs. You'll create [traces][] and [metrics][] and emit
them to the console.
```

Reply:
Bigger suggestion here -- since we're not using the go autoinstrumentation agent (maybe that comes in the future!) I think it's worth rephrasing here to emphasize libraries and custom instrumentation with APIs.

Reply:
Should I also mention the SDK? Is isn't the term "OpenTelemetry APIs confusing (because there is also OpenTelemetry API)?

I will simply change to

```suggestion
You will learn how you can manually instrument a simple application, in
such a way that [traces][] and [metrics][] are emitted to the console.
```

But I am open for further suggestions.

PS. I have created https://github.com/open-telemetry/opentelemetry.io/pull/3308 before this PR and I have forgotten to remove "automatically" 🤦

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3310/commits/9b0888dec9fca709e284ca90540c56fe1705ea42